app/controllers/api/v1/packages_controller.rb

```diff
@@ -312,6 +312,8 @@ def package_record
           @package.inventory_number = inventory_number
           @package.box_id = box_id
           @package.pallet_id = pallet_id
+          @package.stockit_designated_by_id = designated_from_stockit ? current_user_id : nil
```

Comment:
Hi @Darshan-Chauhan It would be good to write instance method in package model and perform this operation in that method. 

Just a suggestion. Correct me if you have any other thoughts on it. :)